### DISCUSSION TITLE
i know tw classes but i havent tw on present project. could i copypaste css code from ur docs?

### DISCUSSION THREAD

**Discussion by warsngl** [tailwindlabs/tailwindcss#10159] [Help]
i know tw classes but i havent tw on present project. could i copypaste css code from ur docs?

**Comment by ProPhycient5** [comment]
@warsngl Please elaborate on your query so that it can be comprehended properly to be resolved.

**Reply by warsngl** [reply]
it would be cool to see ready-made css code for each class in the documentation
